x-pack/examples/reporting_example/public/plugin.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  AppMountParameters,
+  AppNavLinkStatus,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+} from '../../../../src/core/public';
+import { PLUGIN_NAME } from '../common';
+import { SetupDeps, StartDeps } from './types';
+
+export class ReportingExamplePlugin implements Plugin<void, void, {}, {}> {
+  public setup(core: CoreSetup, { developerExamples, ...depsSetup }: SetupDeps): void {
+    core.application.register({
+      id: 'reportingExample',
```

Comment:
Did you have a constants file for this `x-pack/examples/reporting_example/common/index.ts`?

Reply:
Also I see `Symbol('some-id')` in a lot of places, might be good here as well

Reply:
e472c7c4477